src/index.ts

```diff
@@ -121,6 +136,15 @@ const runtime: Runtime = {
     return cacheFiles;
   },
   shouldServe: async (options): Promise<boolean> => {
+    debug(`Requested ${options.requestPath} for ${options.entrypoint}`);
+
+    const VERCEL_RUST_EXPERIMENTAL_ROUTE_MERGE =
+      process.env.VERCEL_RUST_EXPERIMENTAL_ROUTE_MERGE === 'true';
```

Comment:
question: can we detect this without an env var?

Reply:
Yes, but then we might run into unexpected behavior if people used the merged route handler path by accident.

Reply:
How would they use it by accident? 😂  
You mean if we incorrectly detect merged?

Reply:
Right now we use the file `api/vercel/index.rs` for the deployed lambda, this means if we made detection based on that one might opt-in unknowingly by creating this file.